src/prettytable/prettytable.py

```diff
@@ -1476,8 +1496,8 @@ def _set_random_style(self) -> None:
 
         self.header = random.choice((True, False))
         self.border = random.choice((True, False))
-        self._hrules = random.choice((ALL, FRAME, HEADER, NONE))
-        self._vrules = random.choice((ALL, FRAME, NONE))
+        self._hrules = random.choice(list(HRuleStyle._value2member_map_))
+        self._vrules = random.choice(list(VRuleStyle._value2member_map_))
```

Comment:
The order changes a bit and therefore also the output since the seed is kept the same. https://github.com/jazzband/prettytable/pull/327/commits/0c23205e62f5d73424e88eaca3a9a899af422a82

Technically a small behavior change, however as it's for `TableStyle.RANDOM` I think it's fine. The alternative would be to replace each constant with the corresponding enum value which would preserve the order.

Reply:
I think order change is okay too, although `_value2member_map_` isn't very clear. Can we keep the individual enums instead?